go.sum

```diff
@@ -1,2 +1,16 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
I added ‘stretchr/testify’ to get more concise tests. However, I get the feeling it will be frown upon because it brings too many dependencies ?